DemoApp/Shared/DemoUsers.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 import StreamChat
 
-let apiKeyString = ProcessInfo.processInfo.environment["CUSTOM_API_KEY"] ?? DemoApiKeys.frankfurtC1
+let apiKeyString = ProcessInfo.processInfo.environment["CUSTOM_API_KEY"] ?? DemoApiKeys.usEastC6
```

Comment:
Yes, although I was thinking that maybe we can just switch to always using the SwiftUI API Key 🤔 Lately the other environemnt is not very useful. Customers will open the app, go to Threads, and see an error. It might not give the best first impression

Reply:
in that case, push notifications will not work on UIKit, and in general, they are separate SDKs. I would ask the backend to address this and fix the issues on the UIKit environment.

Reply:
I will revert all of this in the final PR then 👍